### ISSUE TITLE
🐛 Bug Report: Could not execute command yarn install

### ISSUE THREAD

**Issue by shaynewei666** [backstage/backstage#16071]
### 📜 Description

I followed the Getting Started but it's failing when running the command .npx @backstage/create-app@latest

### 👍 Expected behavior

Able to successfully install backstage

### 👎 Actual Behavior with Screenshots

![image](https://user-images.githubusercontent.com/124037164/215720218-2d98dbcc-5356-417d-ad95-2f46e17dc65e.png)
![image](https://user-images.githubusercontent.com/124037164/215720247-21f95b1f-7bca-4fac-9f01-35711696d093.png)
![image](https://user-images.githubusercontent.com/124037164/215720272-5aee3370-0068-4d3f-95fb-1ac988edf2bd.png)

### 👟 Reproduction steps

1. Open Terminal

2. Install yarn npm install -g yarn

3. Check yarn successfully installed by running this returns something similar to yarn --version3.2.1

4. Run the command npx @backstage/create-app@latest

5. Notice the error fails with the error above

### 🖥️ Your Environment

centos-release-7-9.2009.1.el7.centos.x86_64
Yarn: 1.22.19
Node: v16.19.0

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

None

**Comment by Elya29** [comment]
Hey, I have the same problem here. I finally created my app from a local clone of the Backstage repository. In the create-app folder (packages/create-app), run the following commands :
`yarn install
yarn backstage-create-app`

Hope it will help !

**Comment by freben** [comment]
This seems to be an environment related problem. Your build doesn't succeed it seems, because of some incompatibility with flags for g++.

Isn't centos basically past end of life? Not sure what versions of libraries and compilers etc that it might have.

If I'm reading things correctly it's trying to build better-sqlite3. You may want to contact the author of that library to see if they can or are willing to support your operating system for builds.

**Comment by Mostafa-Davoodi** [comment]
I faced with the same issue, is there any update?

**Comment by BenjoGreeno** [comment]
I know this is really old and closed but I just figured this out on Ubuntu 24.04 after a bunch of pain and this seemed to be the only place I could find with people faced with the same issue.
The generic Yarn error in my case seemed to stem from isolated-vm not liking Node 18, but upgrading to Node 21 kills Yarn. Node 20 seems to be the sweet spot at the time of writing this as I was able to install isolated-vm via npm. After updating the repo list [here](https://itsfoss.com/upgrade-node-ubuntu/), I then had to completely removed node and re-installed before running `npx @backstage/create-app@latest`
